### PR TITLE
docs(self-hosting): describe updating and scaling

### PR DESCRIPTION
**Pull request by eliobischof**

### CODE REVIEW

File: docs/docs/self-hosting/manage/updating_scaling.md [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+
+For production setups, we recommend that you run  ZITADEL in a highly available mode.
+As all shared state is managed in the database,
+the ZITADEL binary itself is stateless.
+You can easily run as many ZITADEL binaries in parallel as you want.
+If you use [Knative](/docs/self-hosting/deploy/knative)
+or [Google Cloud Run](https://cloud.google.com/run) (which uses Knative, too),
+you can even scale to zero.
+Especially if you use an autoscaler that scales to zero,
+it is crucial that you minimise startup times.
```
Comment (mffap) [inline docs/docs/self-hosting/manage/updating_scaling.md:18]:
```suggestion
it is crucial that you minimize startup times.
```
en-us

File: docs/docs/self-hosting/manage/updating_scaling.md [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+this step can take several minutes.
+When deploying a new ZITADEL version,
+make sure the setup phase runs before you roll out the new `zitadel start` processes.
+The setup phase is executed in subsequent steps
+whereas a new version's execution takes over where the last execution stopped.
+Therefore, configuration changes relevant for the setup phase won’t take effect in regard to the setup execution.
+
+### The Start Phase
+
+The `zitadel start` command assumes the database is already initialised and set up.
```
Comment (mffap) [inline docs/docs/self-hosting/manage/updating_scaling.md:62]:
```suggestion
The `zitadel start` command assumes the database is already initialized and set up.
```

File: docs/docs/self-hosting/manage/updating_scaling.md [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+as the init and setup phases are already separated and executed in dedicated Kubernetes Jobs.
+If you use another orchestrator and want to separate the phases yourself,
+you should know what happens during these phases.
+
+### The Init Phase
+
+The command `zitadel init` ensures the database connection is ready to use for the subsequent phases.
+It just needs to be executed once over ZITADELs full life cycle,
+when you install ZITADEL from scratch.
+Using the configured database admin user, `zitadel init` ensures the following:
```
Comment (mffap) [inline docs/docs/self-hosting/manage/updating_scaling.md:42]: While reading this I got confused by the mention of a specific user. How can I define or run `zitadel init` with a  different user? Do I need to add some parameters to execute as admin user?
Can we link this here or remove the reference to the admin user?

File: docs/docs/self-hosting/manage/updating_scaling.md [hunk 4]
```diff
@@ -0,0 +1,69 @@
...
+You can easily run as many ZITADEL binaries in parallel as you want.
+If you use [Knative](/docs/self-hosting/deploy/knative)
+or [Google Cloud Run](https://cloud.google.com/run) (which uses Knative, too),
+you can even scale to zero.
+Especially if you use an autoscaler that scales to zero,
+it is crucial that you minimise startup times.
+
+## Updating ZITADEL
+
+When updating ZITADEL, the setup job takes care of database migrations.
```
Comment (mffap) [inline docs/docs/self-hosting/manage/updating_scaling.md:22]: Is the setup job `zitadel setup`? I'm not completely sure if I would have to run that command manually or if this is done automatically when using a new version (I assume manual).

**Review by mffap — CHANGES_REQUESTED** [review]
@eliobischof when reading through the docs, I got a bit stuck/confused by two things mentioned in the comments. Can you maybe clarify these in the text? Might be totally trivial, but could be useful to make it explicit. 

Also shouldn't the production checklist link here form the section High Availability?
[/docs/self-hosting/manage/production#high-availability](https://zitadel.com/docs/self-hosting/manage/production#high-availability)

--> Maybe we can then also fix the link to kubernetes on that page

File: docs/docs/self-hosting/manage/updating_scaling.md (hunk 3 above)
Comment (eliobischof) [reply docs/docs/self-hosting/manage/updating_scaling.md]: The admin user is important and has to exist already. Is that more clear with the latest commit?

File: docs/docs/self-hosting/manage/updating_scaling.md (hunk 4 above)
Comment (eliobischof) [reply docs/docs/self-hosting/manage/updating_scaling.md]: I added a TL;DR in the beginning where I tried to make that clear

**Review by mffap — APPROVED** [review]
Ok, that was definitely worth the extra time. Thanks, is understandable - even to me ;)

**Pull request merged**